3rdparty/libconfig/extra/gen/clangwarnings.patch

```diff
@@ -22,15 +22,15 @@ index 60882db..8b8af33 100644
  #include <errno.h>
  #include <stdlib.h>
 +#include <assert.h>
- 
+
  /* end standard C headers. */
- 
+
 @@ -1702,6 +1703,8 @@ static int yy_get_next_buffer (yyscan_t yyscanner)
-             libconfig_yy_create_buffer(yyin,YY_BUF_SIZE ,yyscanner);
+             yy_create_buffer( yyin, YY_BUF_SIZE , yyscanner);
```

Comment:
```suggestion
             yy_create_buffer(yyin, YY_BUF_SIZE, yyscanner);
```

Reply:
this is a git patch so it must match what is in the source, and the libconfig generator just happens to generate scanner.c this way

Reply:
Sorry, my fault.